crates/egui/src/input_state/touch_state.rs

```diff
@@ -60,6 +60,9 @@ pub struct MultiTouchInfo {
     /// Note 2: Just increasing the physical pressure without actually moving the finger may not
     /// necessarily lead to a change of this value.
     pub force: f32,
+
+    /// Center position of the current gesture (average of all touch points).
+    pub center_pos: Pos2,
```

Comment:
nit: I think this should come earlier in the struct. Putting it last means it is the last thing users will see in the docs.

Reply:
Moved it below start_pos